src/analysisd/decoders/syscollector.c

```diff
@@ -1192,8 +1198,8 @@ int decode_port( Eventinfo *lf, cJSON * logJSON,int *socket) {
         msg_type = cJSON_GetObjectItem(logJSON, "type")->valuestring;
 
         if (!msg_type) {
-            merror("Invalid message. Type not found.");
-            goto end;
+            merror("Invalid message. Type not found."); // LCOV_EXCL_LINE
+            goto end;                                   // LCOV_EXCL_LINE
```

Comment:
I would normalize this message using the approach of the one above by adding the function name. The same for all cases

Reply:
This refactoring exceeds the scope of the issue.

Reply:
ok